src/lib/rxfill.ts

```diff
@@ -0,0 +1,40 @@
+import { Router, Request } from 'express';
+import { remsCaseCollection } from '../fhir/models';
+const router = Router();
+
+router.post('/ncpdp', async (req: Request) => {
```

Comment:
Should this router go in a ncpcp script file since it doesn't just do rxfill? Perhaps put the files in a ncpdp folder and name the file with the router 'script' ?